src/core/ext/filters/client_channel/client_channel_factory.c

```diff
@@ -43,14 +43,15 @@ grpc_channel* grpc_client_channel_factory_create_channel(
 }
 
 static void* factory_arg_copy(void* factory) {
-  grpc_client_channel_factory_ref(factory);
+  grpc_client_channel_factory_ref((grpc_client_channel_factory*)factory);
   return factory;
 }
 
 static void factory_arg_destroy(grpc_exec_ctx* exec_ctx, void* factory) {
   // TODO(roth): Remove local exec_ctx when
```

Comment:
Not really related to this PR, but it looks like this TODO has already been taken care of, so please remove it as long as you're in here.

Reply:
will do

Reply:
done